glue.tf

```diff
@@ -0,0 +1,203 @@
+resource "aws_glue_catalog_table" "this" {
+  provider      = aws.glue
+  count         = var.access_logs_enabled ? 1 : 0
+  name          = "${module.this.namespace}_${replace((var.name), "-", "_")}"
+  database_name = "${module.this.environment}_alb_logs"
+  table_type    = "EXTERNAL_TABLE"
+
+  partition_keys {
+    name = "day"
```

Comment:
can we rename this to date instead?

Reply:
i'd like to stick with day here to keep it consistent with AWS docs